apps/server/lib/lexical/server/boot.ex

```diff
@@ -78,8 +79,8 @@ defmodule Lexical.Server.Boot do
   defp load_app_modules(app_name) do
     Application.ensure_loaded(app_name)
 
-    with {:ok, modules} <- :application.get_key(app_name, :modules) do
-      Enum.each(modules, &Code.ensure_loaded!/1)
+    with {:ok, modules} <- Application.spec(app_name, :modules) do
```

Comment:
I think the reason this is faster is that this block never executes. `Application.spec` returns either the list of modules or nil, which causes this arm of the with to not match, and `Code.ensure_all_loaded!` won't run.

Reply:
good catch.